Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -232,6 +232,11 @@ extension BrowserViewController: WKNavigationDelegate {
             request = pendingRequests.removeValue(forKey: url.absoluteString)
         }
         
+        // We can only show this content in the web view if this web view is not pending
+        // download via the context menu.
+        let canShowInWebView = navigationResponse.canShowMIMEType && (webView != pendingDownloadWebView)
+        let forceDownload = webView == pendingDownloadWebView
```

Comment:
What was the reason for relocating this logic above the url code below?

--
(if this was moved over from FF, ignore the comment below)
I know this existed before, but since this is just inverted of part of the logic above maybe:
```
        let forceDownload = webView == pendingDownloadWebView
        let canShowInWebView = navigationResponse.canShowMIMEType && !forceDownload
```

Reply:
moved from FF